types/node-telegram-bot-api/index.d.ts

```diff
@@ -299,17 +303,13 @@ declare namespace TelegramBot {
 
     interface RestrictChatMemberOptions {
         until_date?: number | undefined;
-        can_send_messages?: boolean | undefined;
-        can_send_media_messages?: boolean | undefined;
-        can_send_polls?: boolean | undefined;
-        can_send_other_messages?: boolean | undefined;
-        can_add_web_page_previews?: boolean | undefined;
-        can_change_info?: boolean | undefined;
-        can_invite_users?: boolean | undefined;
-        can_pin_messages?: boolean | undefined;
```

Comment:
Are these options still available if use_independent_chat_permissions is set to true?

Reply:
as said [here](https://core.telegram.org/bots/api#restrictchatmember) if `use_independent_chat_permissions` true: some permissions doesn't inherit from other permissions so we can use all this permissions only when `use_independent_chat_permissions` seted to true

Reply:
Seems like the telegram bot api has introduced a beaking change here. Anyway this part LGTM now